### DISCUSSION TITLE
Docs Navs Sticky Scroll feature is available

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#791] [General]
# Docs Navs Sticky Scroll feature is available

Visual Studio Code sticky scroll feature is really cool, which making the current directory hierarchy clear at a glance. Inspired by it, we implemented a similar feature for document layout's navigations.

https://hbstack.dev/news/2024/09/docs-navs-sticky-scroll-feature-is-available/

**Comment by LITUATUI** [comment]
Really cool feature, thanks!

I've been look for good themes for documentation websites, I'll definitely try it.

**Reply by razonyang** [reply]
I'm glad you like it, please feel free to report any issues you find.
